.env.example

```diff
@@ -17,4 +17,7 @@ MAIL_PORT=2525
 MAIL_USERNAME=null
 MAIL_PASSWORD=null
 
-MOLLIE_API_KEY=test_3RAxAtCZAZh5pZAjUcf6U89WH6DNVr
\ No newline at end of file
+MOLLIE_API_KEY=test_3RAxAtCZAZh5pZAjUcf6U89WH6DNVr
+
+BANKING_INFORMATION="NL68 TRIO 0198 4197 83 t.n.v. Vereniging Anderwijs te Utrecht"
+ADDRESS_INFORMATION="Vereniging Anderwijs\nT.a.v. de penningmeester\nPostbus 13228\n3507 LE Utrecht"
```

Comment:
Beetje scope creep, maar nice refactor 👍

Reply:
Oh oeps ik denk dat dat een artefact is van changes uit die mailable test

Reply:
Ik haal het er wel uit, dat hoort hier niet. Dit hoort in de mailable unit test pr